bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+        400:
+          description: Invalid address format.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/errors/hash/{transaction_hash}:
+    get:
+      summary: Returns wrap errors for a specific transaction hash.
+      description: Retrieves wrap errors generated during processing of a transaction with the given hash.
+      tags:
+        - Wrap
+      operationId: getWrapErrorsByHash
+      parameters:
+        - $ref: '#/components/parameters/TransactionHash'
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+      responses:
+        200:
+          description: Wrap errors associated with the specified transaction hash.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapError'
+        400:
+          description: Invalid transaction hash format.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/prepare:
+    post:
+      summary: Prepares a wrap transaction.
+      description: |
+        Estimates the outcome of a wrap operation, including conversion results and fees.
+        This endpoint simulates how many wrapped tokens the user will receive based on the bridge’s
+        current configuration, fee structure, and mode.
+      tags:
+        - Wrap
+      operationId: prepareWrap
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PrepareRequest'
+      responses:
+        200:
+          description: Estimated wrap transaction details, including fees and net received amount.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PrepareResponse'
+        400:
+          description: Wrap request parameters are invalid or violate bridge limits.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal bridge error during preparation.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /unwrap/requests:
```

Comment:
The wrap and unwrap endpoints are pretty symmetrical, so all my previous comments apply to unwrap operations too.